templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -958,6 +977,88 @@ bool IKFastKinematicsPlugin::getPositionIK(const geometry_msgs::Pose &ik_pose,
   return false;
 }
 
+// Used when there are no redundant joints - aka no free params
+bool IKFastKinematicsPlugin::getPositionIK(const std::vector<geometry_msgs::Pose> &ik_poses,
+                                           std::vector< std::vector<double> >& solutions,
+                                           kinematics::KinematicsResult& result,
+                                           const kinematics::KinematicsQueryOptions &options) const
+{
+  ROS_DEBUG_STREAM_NAMED("ikfast","getPositionIK with multiple solutions");
+
+  if(!active_)
+  {
+    ROS_ERROR("kinematics not active");
+    return false;
+  }
+
+  std::vector<double> vfree(free_params_.size());
+  const std::vector<double> ik_seed_state(num_joints_);
+  for(std::size_t i = 0; i < free_params_.size(); ++i)
+  {
+    int p = free_params_[i];
+    ROS_ERROR("%u is %f",p,ik_seed_state[p]);  // DTC
+    vfree[i] = ik_seed_state[p];
```

Comment:
The redundancy sampling could be turned off by setting the appropriate value in the [KinematicsQueryOption](https://github.com/jrgnicho/moveit_core/blob/ikplugin_changes/kinematics_base/include/moveit/kinematics_base/kinematics_base.h#L103) structure, in which case it would behave the same.  In fact, it defaults to no discretization.